source/_integrations/sms.markdown

```diff
@@ -97,13 +98,17 @@ Need to unlock it using [this guide](http://blog.asiantuntijakaveri.fi/2015/07/c
 
 Search in the [Gammu database](https://wammu.eu/phones/) for modems with AT connection.
 
-### Huawei modems on Raspberry Pi (and similar) devices
+### Huawei/ZTE modems (and similar) devices - NOT applicable for users of Home Assistant OS, Container or Supervised.
```

Comment:
That is not correct. The previous instructions did work, now it doesn't apply anymore.
This PR renders the documentation incorrect for our largest user group (while previously it was correct).

Reply:
I'm a user of a debian instalation and the problem persists as `Bullseye` release, not being exclusive of the debian distro (raspbian) for Raspberry PI. The issue seems to exist in all debian-like distros for some unknown reason.